.github/workflows/build.yml

```diff
@@ -50,4 +49,48 @@ jobs:
       - name: Upload Code Coverage
         uses: codecov/codecov-action@v1.5.0
         with:
-          flags: dotnet
\ No newline at end of file
+          flags: dotnet
+
+  build-generator:
```

Comment:
>build [](http://example.com/codeflow?start=2&length=5)

do we need official build changes?

Reply:
I don't believe so - the official build just triggers the nupkg build (which is unchanged)

Reply:
Worth building all the things regardless? (I'm good either way.)

Reply:
I'm fine with the official build just building the releasable assets for now - CI on the main branch and on PRs should catch any regressions in the generator code.